corehq/apps/hqwebapp/static/hqwebapp/js/sso_inactivity.js

```diff
@@ -0,0 +1,9 @@
+/*
+ * Handles communication about the status of SSO login after the inactivity
+ * timer has requested a re-login
+ */
+hqDefine('hqwebapp/js/sso_inactivity', [], function () {
+    localStorage.setItem('ssoInactivityMessage', JSON.stringify({
+        isLoggedIn: true,
+    }));
```

Comment:
Do localStorage messages require JSON, or dictionaries? Especially given that `checkIfSsoMessageReceivedFromExternalTab` just deletes the entire `ssoInactivityMessage` after processing, it doesn't seem like any value matters except `isLoggedIn` -- so could we simplify `ssoInactivityMessage` to just be the boolean value for `isLoggedIn`?

Reply:
it's not required, but also there's no harm and it makes it clearer what it's looking for tbh.

Reply:
Just nitpicking, but why can't you just do:
`localStorage.setItem('isLoggedIn', true)`? If we only ever store one variable, it seems clearest to just have a scalar variable.